Common/OSVersion.cpp

```diff
@@ -7,6 +7,7 @@
 #endif
 
 #include <cstdint>
+#include <vector>
```

Comment:
Why was this added here?

Reply:
Like the commit message for this change says, libretro windows builds won't compile without it. 1103acd added a reference to vector without adding the include. I'm honestly not certain how your standalone builds compile currently.

Reply:
We include vector in stdafx.h.

-[Unknown]

Reply:
Libretros buildbot uses the makefiles in the libretro directory to build right now. Which to my knowledge, does not use visual studio and thus doesn't use stdafx. And until the before mentioned commit, this has worked. Is it much of a problem to list includes used by a source file in the source file?

Reply:
Alright, fair enough.